src/main/resources/application.yaml

```diff
@@ -8,6 +8,10 @@ spring:
   jpa.hibernate.ddl-auto: validate
 liquibase.change-log: classpath:db/changelog/db.changelog.xml
 
+server.compression:
+  enabled: true
+  mime-types: application/atom+xml,text/html,text/xml,text/plain,text/css,text/javascript,application/javascript
+ 
```

Comment:
`text/xml,text/plain,text/css,text/javascript,application/javascript` is the default value for `mime-types`. When adding a custom `application/atom+xml` value to the list the defaults are overridden so we have to provide it explicitly.